### ISSUE TITLE
🕷️ 🌎Feed crawler

### ISSUE THREAD

**Issue by n0str** [itmo-wad/WAD-Course-ITMO#2]
# Feed crawler

Feed crawler – service which posts the best (under multiple criteria) news from media services and social networks.

**Problem**: There is too much information on the Internet. You never have time to read your feeds in twitter, VK, facebook, reddit, etc.

**Solution**: Service which will chose the best news which fits your taste.

**Simplified cases**: 
* A person wants to know about recent news in tech. He goes to special Telegram group and gets 10 best articles (by score) every day.
* Another person likes something funny. He goes to a telegram group where he can get 10 best (by likes count) memes from reddit, pikabu and tjournal.

**References**:
 * https://vk.com/r_all
 * https://vk.com/public55993443

**Comment by lorrod** [comment]
I like this idea because it could be helpful for a lot of people. But I think it needs one more feature that lets users select groups which will be parsed in future and that will not be a Telegram group, because the news will not be personalized, better to do a Telegram bot.

**Additional problems:** 

- Social media often distract us from everyday activities by sending a lot of articles. 

- There is a lot of different social media and it is not comfortable to go from one to another social media

**Case of usage:**

- The user goes into Telegram bot and choose groups from social media, then he chooses if it should be the top 10 news or all the news from all groups which user selected before. Articles will be shown on his criteria by pressing update button

**Comment by MenardPhiri** [comment]
Answering 5. (Write at least one comment why you do not like one project. Comment should include a detailed explanation of why.)

Ironically, its more easy to see the good than the bad in almost all your project ideas. Your projects are well thought through, and all have clear goals, and addressing some issues in tech. However, for instance, this project does not ressonate well with me, largely because It could not be a full product on its own(as it would just be a service to another product/non stand alone product), and it has extreme potential to errode my privacy. Its behavour profile building of my online presence as the solution assumes (Solution: Service which will chose the best news which fits your taste), much as we know its for good reasons, has the ability to be abused security wise, and could raise interesting privacy issues that as developer I could not be willing to guaratee. This does not in anyway, prempt measures that could address all such, but just dries my passion, zeal and energy to learn the skills match for delivery, as I am blided by less value, impact pontetial, interoperability prospects.

Trivially, the project description seems the solution being proposed, and simplified cases are not in much sync, much as it could be undersight. The former assumes service personalisation, while the latter assumes just ratings that could be far from my taste. As such, my security privacy highlight, could only hold for the former.

**Comment by lorrod** [comment]
Я согласен с тобой, menardPhiti, очень сложно будет угодить пользователю с персонализированным подбором

**Comment by lorrod** [comment]
I agree with you, MenardPhiri, it will be very difficult to personalize news for every user
